src/helpers/trainingContracts.js

```diff
@@ -78,7 +79,7 @@ exports.formatCourseForTrainingContract = async (course, vendorCompany, payloadP
   const trainees = course.trainees
     .filter(t => UtilsHelper.areObjectIdsEquals(course.companies[0]._id, traineesCompany[t._id]));
   const price = (course.prices || []).find(p => UtilsHelper.areObjectIdsEquals(p.company, course.companies[0]._id));
-  const totalPrice = price ? Number(get(price, 'global', 0) + get(price, 'trainerFees', 0)) : 0;
+  const totalPrice = price ? Number(NumbersHelper.add(get(price, 'global', 0), get(price, 'trainerFees', 0))) : 0;
```

Comment:
Si tu utilises bigNumber, c'est pas necessaire de conserver number si ?

Reply:
en soit c'est pas nécessaire absolument mais c'est pas homogénéité, pour avoir des nombres partout et pas parfois des strings et parfois des nombres (bignumber renvoie une string)